tests/integration/render.int.js

```diff
@@ -217,7 +217,32 @@ describe('Full dummy app render', () => {
 			};
 			return server
 				.inject(request)
+				.then(response => {
+					expect(response.statusCode).toBe(500);
+				})
+				.then(() => server.stop())
+				.catch(err => {
+					server.stop();
+					throw err;
+				});
+		}));
+	it('returns 200 OK after a 500 error', () =>
```

Comment:
Nice.

Hopefully this catches other uses of "react-side-effects" where there is potential for a similar error.